cgi/search.pl

```diff
@@ -24,18 +24,18 @@
 use utf8;
 
 use CGI::Carp qw(fatalsToBrowser);
-use CGI qw/:cgi :form escapeHTML/;
-
-use ProductOpener::Config qw/:all/;
-use ProductOpener::Store qw/:all/;
-use ProductOpener::Index qw/:all/;
-use ProductOpener::Display qw/:all/;
-use ProductOpener::Users qw/:all/;
-use ProductOpener::Products qw/:all/;
-use ProductOpener::Food qw/:all/;
-use ProductOpener::Tags qw/:all/;
+use CGI       qw/:cgi :form escapeHTML/;
+
+use ProductOpener::Config        qw/:all/;
+use ProductOpener::Store         qw/:all/;
+use ProductOpener::Index         qw/:all/;
+use ProductOpener::Display       qw/:all/;
+use ProductOpener::Users         qw/:all/;
+use ProductOpener::Products      qw/:all/;
+use ProductOpener::Food          qw/:all/;
+use ProductOpener::Tags          qw/:all/;
 use ProductOpener::PackagerCodes qw/:all/;
-use ProductOpener::Text qw/:all/;
+use ProductOpener::Text          qw/:all/;
```

Comment:
I'm really not keen on this kind of tabular alignment as it is really hard to maintain for a human, and it does not bring much in my opinion.